.github/workflows/docker-mod-builder.yml

```diff
@@ -0,0 +1,119 @@
+name: Docker Mod Builder
+
+on:
+  workflow_call:
+    inputs:
+      BASEIMAGE:
+        required: true
+        type: string
+      MODNAME:
+        required: true
+        type: string
+      ENDPOINT:
+        required: true
+        type: string
+      GITHUB_REPO:
+        required: true
+        type: string
+      MOD_VERSION:
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Dump payload
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+      - name: Set Vars
+        run: |
+          echo "BASEIMAGE=${{ inputs.BASEIMAGE }}" >> $GITHUB_ENV
+          echo "MODNAME=${{ inputs.MODNAME }}" >> $GITHUB_ENV
+          echo "ENDPOINT=${{ inputs.ENDPOINT }}" >> $GITHUB_ENV
+          echo "GITHUB_REPO=${{ inputs.GITHUB_REPO }}" >> $GITHUB_ENV
+          echo "MOD_VERSION=${{ inputs.MOD_VERSION }}" >> $GITHUB_ENV
+      - name: Dump vars
+        run: |
+          echo "BASEIMAGE is ${BASEIMAGE}"
+          echo "GITHUB_REPO is ${GITHUB_REPO}"
+          echo "MODNAME is ${MODNAME}"
+          echo "ENDPOINT is ${ENDPOINT}"
+          echo "MOD_VERSION is ${MOD_VERSION}"
+          echo "github.repository is ${{ github.repository }}"
+          echo "github.event.pull_request.base.ref is ${{ github.event.pull_request.base.ref }}"
+          echo "github.event.pull_request.base.repo.name is ${{ github.event.pull_request.base.repo.name }}"
+      - name: Set up qemu
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
There's an action for the qemu setup you can use
```
      - name: Set up QEMU
        uses: docker/setup-qemu-action@v2
```

Reply:
Done